src/pybind/mgr/cephadm/module.py

```diff
@@ -3082,6 +3083,15 @@ def place(self):
             logger.debug('All hosts: {}'.format(candidates))
             return candidates
 
+        # respect host_pattern
+        if self.spec.placement.host_pattern:
```

Comment:
We may even go one step further and use the `placement` notation in the drivegroups?

``` yaml

name: fooo
placement:
  label: osd
  ...
data_devices:
  all: True
```

That would actually unify the servicespecs with the drivegroups

Reply:
since host_pattern accepts a glob, we probably have to have something in the PlacementSpec that also accepts a glob..

Reply:
Oh, I think that's what you do already :)